ドキュメントの翻訳対応 (#16)
civictechzenchiba/covid19-chiba: .github/CONTRIBUTING_EN.md

# How to contribute

## Comments on Issue and Pull Request
* Please feel free to comment on Issue! We accept new questions and suggestions.
* Please note that issues with improve (improvement proposal) are not always reflected (Especially for the UI system).

> * It would be helpful if you prioritize good first issue / help wanted / bug.

Reviewer: ここは東京版の文が過不足なくよさそうでした。

```suggestion
* We appreciate if you give priority to the issues labelled [good first issue / help wanted / bug]
```

## Getting Started

> * We welcome those who want to make good use of ICT in Chiba Prefecture and participate!

Reviewer: こんな感じでしょうか？
```suggestion
* We welcome any participants who want to improve Chiba with ICT!
```
